README.md

```diff
@@ -437,6 +437,20 @@ or any other nested information.
 
 <img width="500" alt="A refactored DAG showing three sources which are each being referenced by an accompanying staging model" src="https://user-images.githubusercontent.com/30663534/159603703-6e94b00b-07d1-4f47-89df-8e5685d9fcf0.png"> 
 
+### Exposure Materializations
+#### Model
+
+`fct_exposure_materializations` ([source](models/marts/dag/fct_exposure_materializations.sql)) shows each model with a direct relationship to an exposure that is *not* materialized as a table in the warehouse. 
+
+#### Reason to Flag
+
+Models that are referenced by an exposure are likely to be used heavily in downstream systems, and therefore need to be performant when queried. This model highlights instances where the models referenced by exposures are not either a `table` or `incremental` materialization.
```

Comment:
I actually do think a screenshot of the DAG & example from our integration tests would be useful here, just to visualize that we're looking at the materialization of the model directly upstream of the exposure.

Reply:
agreed -- do you think i should try to somehow try to viusally capture tehir materializations? like renaming the nodes to be `view_` or `table_` prefixed?

Reply:
That could work - or just add a note below the screenshot to explain the materialization of each model. The latter might be easier.

Reply:
added screenshot and note